qbertconfig/cli/dispatcher.py

```diff
@@ -30,5 +29,5 @@ def do(self, operation, targets):
     def fetch(self, args):
         cluster_name = args.name if args.name else None
         cluster_uuid = args.uuid if args.uuid else None
-        self.kubeconfig.fetch(self.cloud, cluster_name=cluster_name, cluster_uuid=cluster_uuid)
-        self.kubeconfig.save()
+        self.qbert_config.fetch(cluster_name=cluster_name, cluster_uuid=cluster_uuid)
```

Comment:
what about having `fetch` return a `Kubeconfig` object? that way we can call save directly on it. This would also save us from implementing a `save` wrapper method in `QbertConfig`